templates/huginn/0/docker-compose.yml

```diff
@@ -0,0 +1,114 @@
+version: '2'
+
+services:
+  postgres:
+    image: postgres:9.5
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: ${DB_PASSWORD}
+      POSTGRES_USER: huginn
+      POSTGRES_DB: huginn
+      http_proxy:
+      https_proxy:
+    labels:
+      io.rancher.container.pull_image: always
+
+  huginn-web:
+    image: huginn/huginn-single-process
```

Comment:
Please, use concrete container image tag. Generic tags implicit neither explicit, are not allowed due to are not reproducible.